src/pages/BanquetFacility/BanquetFacility.js

```diff
@@ -21,31 +21,37 @@ function createMarkup(body) {
   return {__html: body};
   }
 
-const BanquetFacility = ({data}) => {
-  if (!data) return null;
+const BanquetFacility = ({ data }) => {
 
-  let {section_one, section_two} = data.banquet_facility_page;
+if (!data) return null;
+
+let { builder } = data;
+let [image_banner, title_subtitle_body, image_title_body_button] = builder;
+
+image_banner = image_banner.image_banner;
+title_subtitle_body = title_subtitle_body.title_subtitle_body;
+image_title_body_button= image_title_body_button.image_title_body_button;
 
   return(
     <Layout>
       <Container>
         <div className="banquet-facility">
           <div className="banquet-facility__triple">
-            <Image className="banquet-facility__image--triple" img={"https://dev.huoa.org/wp-content/uploads/2020/12/banquet-facility-1.webp"} alt=""></Image>
-            <Image className="banquet-facility__image--triple" img={"https://dev.huoa.org/wp-content/uploads/2020/12/banquet-facility-2.webp "} alt=""></Image>
-            <Image className="banquet-facility__image--triple" img={"https://dev.huoa.org/wp-content/uploads/2020/12/banquet-facility-3.webp"} alt=""></Image>
+          {image_banner.map((el, i) =>  {
+            return <Image key={i} className="banquet-facility__image--triple" img={el.url} alt={el.alt}></Image>      
```

Comment:
The naming convention here could be updated / refreshed. Example what is `banquet-facility__triple` doing

Reply:
I agree. I changed all the classes in the banquet facility PR.